packages/components/tabs/src/tabs.tsx

```diff
@@ -71,7 +70,7 @@ export const tabsEmits = {
   'tab-change': (name: TabPanelName) => isPanelName(name),
   edit: (paneName: TabPanelName | undefined, action: 'remove' | 'add') =>
     ['remove', 'add'].includes(action),
-  'tab-remove': (name: TabPanelName) => isPanelName(name),
+  'tab-remove': (name?: TabPanelName) => isPanelName(name),
```

Comment:
After canceling, it will cause an error message on the handleTabRemove side.

One of the methods should be used.
```ts
const handleTabRemove = (pane: TabsPaneContext, ev: Event) => {
    if (pane.props.disabled) return
   // if (pane.props.disabled || isUndefined(pane.props.name)) return
    ev.stopPropagation()
    emit('edit', pane.props.name, 'remove')
    emit('tab-remove', pane.props.name)
   // !isUndefined(pane.props.name) && emit('tab-remove', pane.props.name)
  }
```

Reply:
same with setCurrentName

`if (pane.props.disabled || isUndefined(pane.props.name)) return`